### PR TITLE
Feature/dev 100/voice chat

### PR DESCRIPTION
**Pull request by PAKovalev**

### CODE REVIEW

File: packages/ui-kit-storybook/src/static/styles/variables.css [hunk 1]
```diff
@@ -2,6 +2,8 @@
   /** Primary colors **/
   --white: #ffffff;
   --black: #000000;
+  --brawn: #bb7851;
```
Comment (dmitry-yudakov) [inline packages/ui-kit-storybook/src/static/styles/variables.css]: typo, should be brown

File: packages/ui-kit-storybook/src/molecules/VoiceUser/VoiceUser.styled.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import {rgba} from 'polished';
+
+const BROWN = '#bb7851';
+const YELLOW = '#eae71b';
```
Comment (dmitry-yudakov) [inline packages/ui-kit-storybook/src/molecules/VoiceUser/VoiceUser.styled.ts:5]: it's set in 2 places, here and in css - maybe just here is good enough.

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/ui-kit-storybook/src/static/styles/variables.css (hunk 1 above)
Comment (PAKovalev) [reply packages/ui-kit-storybook/src/static/styles/variables.css]: Yeah, fixed.

File: packages/ui-kit-storybook/src/molecules/VoiceUser/VoiceUser.styled.ts (hunk 2 above)
Comment (PAKovalev) [reply packages/ui-kit-storybook/src/molecules/VoiceUser/VoiceUser.styled.ts:5]: Let leave both places for now. Css-const will be used also, I think.

**Pull request merged**